addons/website_forum/tests/test_forum.py

```diff
@@ -1,12 +1,13 @@
 # -*- coding: utf-8 -*-
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
-
 from .common import KARMA, TestForumCommon
+from odoo import http
+from odoo.addons.http_routing.models.ir_http import slug
 from odoo.exceptions import UserError, AccessError
+from odoo.tests import HttpCase
 from odoo.tools import mute_logger
 from psycopg2 import IntegrityError
 
-
```

Comment:
You may keep this line (unnecessary diff anyway)

Reply:
Oops my bad, I didn't see that